lib/vertica/row.rb

```diff
@@ -13,23 +15,44 @@ def fetch(name_or_index)
     @values.fetch(column_index(name_or_index))
   end
 
-  def inspect
-    "<Vertica::Row#{@values.inspect}>"
-  end
-
   alias_method :[], :fetch
 
+  # Returns an array representation of the row. The values will
+  # be ordered like the order of the fields in the {#row_description}.
+  # @return [Array]
   def to_a
-    @values.to_a
+    @values
   end
 
+  # Returns a hash representation of this rows, using the name of the
+  # fields as keys.
+  # @param symbolize_keys [true, false] Set to true to use symbols instead
+  #   of strings for the field names.
+  # @return [Hash]
   def to_h(symbolize_keys: false)
     @row_description.inject({}) do |carry, column|
       key = symbolize_keys ? column.name.to_sym : column.name
-      carry.merge(key => fetch(column.name))
+      carry[key] = fetch(column.name)
+      carry
```

Comment:
How do you feel about `#tap` instead of these two lines? I know not everybody loves it, but `tap` has grown on me.
```
carry.tap { |c| c[key] = fetch(column.name) }
```